File: problem896.py
class Solution:
    def isMonotonic(self, nums: List[int]) -> bool:
        x = nums[0]
        flag=0
        for n in nums[1:]:
            if n<x:
                flag = 1
                break
            x=n
            
        if flag==1:
            x=nums[0]
            for n in nums[1:]:
                if n>x:
                    return False    
                x=n

        return True

File: problem896.cs
public class Solution {
    public bool IsMonotonic(int[] nums) {
        int store=0;
        
        for(int i=0; i<nums.Length-1; i++)
        {
            int c = nums[i].CompareTo(nums[i+1]);
            if(c!=0)
            {
                if(c!=store && store!=0)
                {
                    return false;
                }
                store=c;
            }
        }
        
        return true;
        
    }
}